pkg/resourceinterpreter/customizedinterpreter/customized.go

```diff
@@ -32,6 +32,107 @@ type CustomizedInterpreter struct {
 	clientManager *webhookutil.ClientManager
 }
 
+// GetReplicas returns the desired replicas of the object as well as the requirements of each replica.
+// return matched value to indicate whether there is a matching hook.
+func (e *CustomizedInterpreter) GetReplicas(object *unstructured.Unstructured) (replica int32, requires *workv1alpha2.ReplicaRequirements, err error) {
+	var response *webhook.ResponseAttributes
+	var matched bool
+	response, matched, err = e.interpret(context.TODO(), &webhook.RequestAttributes{
```

Comment:
Perhaps the `interpret` method can be further refactored to remove the `matched` return value.

Other parts look good to me.

Reply:
An error can be returned when no matching hook is found.

Reply:
Done!